### ISSUE TITLE
Add Dark Mode

### ISSUE THREAD

**Issue by SwanandD121** [bishalde/QR-Code-Generator#14]
Currently, the project lacks a dark mode feature, which is becoming increasingly essential for user comfort and accessibility. Adding a dark mode option will enhance the user experience, especially for those who prefer a darker interface or are using the application in low-light environments.

**Expected Behavior:**
Upon enabling dark mode, the application's interface should switch to a darker color scheme, providing better readability and reducing eye strain in dimly lit environments.

**Proposed Solution:**
Implement a toggle switch or an option in the application settings to enable dark mode. Upon enabling dark mode, the background color of the interface should change to a darker shade, and text and UI elements should adjust accordingly for improved visibility.

**Additional Considerations:**

- Ensure that the dark mode implementation adheres to accessibility guidelines and provides adequate contrast for readability.
- Consider providing users with the option to automatically switch between light and dark mode based on system preferences.
- Test the dark mode functionality across different devices and screen sizes to ensure consistent behavior and appearance.

**Comment by SwanandD121** [comment]
I have checked the components of your website and have a clear idea on how to implement the dark mode. I have experience with UI/UX design and frontend development, and I believe I can deliver an accessible dark mode implementation. Could I please be assigned to this issue? Looking forward to contributing! Thank you.

Best regards,
Swanand

**Comment by shreyanssatpute** [comment]
I am understanding the specified requirements and i am eager to contribute to this project under GSSoC'24 I have worked closely such kind of project and I believe that i could deliver the required changes as per your expectations  . I request you to please assign me this issue.
Thank You,
Shreyans

**Comment by Ashutosh-0506** [comment]
Hey there! I'm Ashutosh Kumar, a full-stack MERN web developer. With experience in numerous successful MERN stack  projects, I excel in  HTML, CSS,JavaScript,C, C++,jQuery, Node.js, React.js, Bootstrap, Tailwind, Express, MongoDB, API,GraphQL,WebRTC. 
You can find me on LinkedIn : https://www.linkedin.com/in/ashutosh-kumar-60b262251/ 
 GitHub : https://github.com/Ashutosh-0506. 
Looking forward to contributing to GSSOC 2024!

**Comment by sri766** [comment]
Hello @Ashutosh-0506 Can you assign this to me. i have a experience in MERN Stack and Devops. Here is my portfolio for your referrence where i implemented dark mode 
link: https://srisanth.vercel.app/

**Comment by sri766** [comment]
Hello @SwanandD121 Can you assign this to me. i have a experience in MERN Stack and Devops. Here is my portfolio for your referrence where i implemented dark mode.
link: https://srisanth.vercel.app/

**Comment by bishalde** [comment]
Thanyou @SwanandD121  for finding this issue.
Sure, you can proceed with it.!

git fetch origin
git checkout darkmode

I have created a new branch, push the changes in that branch!
And feel free to give this Repository a STAR⭐ and follow back 😊.

**Comment by SwanandD121** [comment]
> Thanyou @SwanandD121 for finding this issue. Sure, you can proceed with it.!
> 
> git fetch origin git checkout darkmode
> 
> I have created a new branch, push the changes in that branch! And feel free to give this Repository a STAR⭐ and follow back 😊.

Thanks for assigning the issue to me!
Also please add the **level** label to the issue.

**Comment by kritikasharma4** [comment]
I have earlier implemented light and dark modes in other projects of mine and want to contribute to this project by working on this issue, 
will you please assign this issue to me
Thankyou!

**Comment by Rishitatripathi** [comment]
I would like to work on this issue, Please assign it to me under GSSOC 2024
